src/__tests__/addremove.test.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @jest-environment jsdom
+ */
+// import { newTodo } from '../index.js';
+
+document.body.innerHTML = `
+    <div class="row">
+            <ul class="todo-list">
+            </ul>
+    </div> `;
+
+let Array;
+let rest;
+
+function newTodo(e) {
+  // e.preventDefault();
+  const newTask = e;
+  const todos = JSON.parse(localStorage.getItem('todos') || '[]');
+
+  const newTodo = {
+    description: newTask,
+    completed: false,
+    id: todos[todos.length - 1] ? todos[todos.length - 1].id + 1 : todos.length + 1,
+  };
+
+  // document.getElementById('new-task').value = '';
```

Comment:
- [ ] please remove commented code to keep your code clean

Reply:
**Thank you @tufoinnkuo10 for reviewing our codes :+1:**